### PR TITLE
CLDC 1465 Add beds question to saleslog

### PR DESCRIPTION
**Pull request by Ted-U**

### CODE REVIEW

File: app/models/form/sales/subsections/setup.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def pages
       Form::Sales::Pages::DiscountedOwnershipType.new(nil, nil, self),
       Form::Sales::Pages::OutrightOwnershipType.new(nil, nil, self),
       Form::Sales::Pages::JointPurchase.new(nil, nil, self),
+      Form::Sales::Pages::PropertyNumberOfBedrooms.new(nil, nil, self),
```
Comment (kosiakkatrina) [inline app/models/form/sales/subsections/setup.rb]: I think this is meant to be in Property information subsection, I think you'd need to create a new subsection, new section and add it to a form in the FormHandler

Comment (dushan-madetech) [reply app/models/form/sales/subsections/setup.rb]: It seems you've created the subsection but you still need to remove this line from here

File: app/models/form/sales/pages/property_number_of_bedrooms.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    @description = ""
+    @subsection = subsection
+  end
+
+  def questions
+    @questions ||= [
+      Form::Sales::Questions::PropertyNumberOfBedrooms.new(nil, nil, self),
+    ]
+  end
+end
```
Comment (dushan-madetech) [inline app/models/form/sales/pages/property_number_of_bedrooms.rb]: I think this should be moved into the property folder you've created

File: spec/models/form/sales/property/pages/property_number_of_bedrooms_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Form::Sales::Pages::PropertyNumberOfBedrooms, type: :model do
```
Comment (dushan-madetech) [inline spec/models/form/sales/property/pages/property_number_of_bedrooms_spec.rb]: This should be `RSpec.describe Form::Sales::Property::Pages::PropertyNumberOfBedrooms`

File: spec/models/form/sales/property/questions/property_number_of_bedrooms_spec.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Form::Sales::Questions::PropertyNumberOfBedrooms, type: :model do
```
Comment (dushan-madetech) [inline spec/models/form/sales/property/questions/property_number_of_bedrooms_spec.rb]: This should be `RSpec.describe Form::Sales::Property::Questions::PropertyNumberOfBedrooms`

File: app/models/form/sales/questions/property_number_of_bedrooms.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class Form::Sales::Questions::PropertyNumberOfBedrooms < ::Form::Question
```
Comment (dushan-madetech) [inline app/models/form/sales/questions/property_number_of_bedrooms.rb]: This should be `Form::Sales::Property::Questions::PropertyNumberOfBedrooms`

File: app/models/form/sales/pages/property_number_of_bedrooms.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+class Form::Sales::Pages::PropertyNumberOfBedrooms < ::Form::Page
```
Comment (dushan-madetech) [inline app/models/form/sales/pages/property_number_of_bedrooms.rb]: When you move this to the property folder this should be `Form::Sales::Property::Pages::PropertyNumberOfBedrooms`

**Review by dushan-madetech — APPROVED** [review]
Looks fine to me now that the suggestions have been implemented 👍

**Pull request merged**